bindings/wasm/native/tests/client.js

```diff
@@ -0,0 +1,183 @@
+const { ClientBuilder } = require('../node')
+const { assertAddress, assertMessageId, assertMessageWrapper } = require('./assertions')
+const assert = require('assert')
+
+const seed = '256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2'
+
+async function test() {
+
+  const client = await new ClientBuilder()
+    .node('https://api.thin-hornet-2.h.chrysalis-devnet.iota.cafe/')
```

Comment:
This one aint working at all :D
`failed, reason: getaddrinfo EAI_AGAIN`

Reply:
That should be a node from the new testnet/devnet which will replace the current one, worked a few days ago :(